src/controller/editingcontroller.js

```diff
@@ -23,6 +23,7 @@ import {
 } from '../conversion/model-selection-to-view-converters';
 
 import EmitterMixin from '@ckeditor/ckeditor5-utils/src/emittermixin';
+import '../../theme/theme.scss';
```

Comment:
If it really worked, we could get rid of nightmares like https://github.com/ckeditor/ckeditor5-link/blob/master/theme/theme.scss#L4. Waiting for confirmation from Reinmar, then I'm gonna create an umbrella issue in [ckeditor5](https://github.com/ckeditor/ckeditor5) because it applies to the entire codebase.